### ISSUE TITLE
Alter space-unary-ops to account for nonwords

### ISSUE THREAD

**Issue by BronsonQuick** [humanmade/coding-standards#15]
Currently we have:

```
space-unary-ops:
  - error
  - words: true
    nonwords: true
    overrides:
      ++: false
      --: false
```
and we should have:
```
space-unary-ops:
  - error
  - words: true
    nonwords: false
```

**Comment by rmccue** [comment]
This needs to add back the override for `!`

**Comment by rmccue** [comment]
This was fixed in #19.
